.github/workflows/cd.yml

```diff
@@ -0,0 +1,239 @@
+        release: edge,beta
+    
+  deploy-snap-arm64:
+    needs: [test-ubuntu, test-windows, test-macos]
+    name: Deploy ARM64 Snap
+    runs-on: ubuntu-latest
+    steps:
+    - uses: docker/setup-qemu-action@v1
+    - uses: actions/checkout@v2
+    - name: Set Up Source
+      run: rsync --filter=":- .gitignore" -r ./ pkg/snap/solvespace-snap-src
+    - name: Build Snap
+      id: build
+      uses: diddlesnaps/snapcraft-multiarch-action@v1
+      with:
+        path: pkg/snap
+        architecture: arm64
+    - name: Upload & Release to Edge
+      if: github.event_name == 'push'
+      uses: snapcore/action-publish@v1
+      with:
+        store_login: ${{ secrets.SNAPSTORE_LOGIN }}
+        snap: ${{ steps.build.outputs.snap }}
+        release: edge
+    - name: Upload & Release to Beta + Edge
+      if: github.event_name == 'release'
+      uses: snapcore/action-publish@v1
+      with:
+        store_login: ${{ secrets.SNAPSTORE_LOGIN }}
+        snap: ${{ steps.build.outputs.snap }}
+        release: edge,beta
+  
+  update_edge_release:
+    name: Update Edge Release
+    needs: [build-release-windows, build-release-windows-openmp, build-release-macos]
+    if: always() && github.event_name == 'push'
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+    - name: Delete Old Edge Release
+      uses: dev-drprasad/delete-tag-and-release@v0.1.2
+      with:
+        delete_release: true
+        tag_name: edge
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Create New Edge Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: edge
+        release_name: Edge
+        prerelease: true
+        body: ${{ github.event.head_commit.message }}
+
+  upload-release-assets:
+    name: Upload Release Assets
+    needs: [update_edge_release]
+    if: always()
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download All Workflow Artifacts
+      uses: actions/download-artifact@v2
+    - name: Get Release Upload URL
+      id: get_upload_url
+      env:
+        event_name: ${{ github.event_name }}
+        event: ${{ toJson(github.event) }}
+        edge_upload_url: ${{ needs.update_edge_release.outputs.upload_url }}
+      run: |
+        if [ "$event_name" = "release" ]; then
+          upload_url=$(echo "$event" | jq -r ".release.upload_url")
+        else
+          upload_url="$edge_upload_url"
+        fi
+        echo "::set-output name=upload_url::$upload_url"
+        echo "Upload URL: $upload_url"
+    - name: Upload solvespace.exe
+      uses: actions/upload-release-asset@v1
+      continue-on-error: true
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.get_upload_url.outputs.upload_url }}
+        asset_path: solvespace.exe/solvespace.exe
```

Comment:
Do these asset_paths make sense? they look different to what we had before, so not sure if it's a mistake or not...

Reply:
Ah, upload/download-artifact, guess this is fine...

Reply:
This makes the filenames look nicer in the artifacts overview. We could do `win-release/solvespace.exe`, but then it shows as `win-release`.